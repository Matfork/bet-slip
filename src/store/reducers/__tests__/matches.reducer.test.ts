import { MatchesActionCreators } from '../../actions/action-creators/matches.actionCreator';
import matchesReducer, { initialState } from '../matches.reducer';

describe('Matches Reducer', () => {
  it('should match snapshot', () => {
    const state = matchesReducer(initialState, {
      type: 'other',
    } as any);

    expect(state).toMatchSnapshot();
  });

  it('should perform GetMatches action', () => {
    const state = matchesReducer(initialState, {
      type: MatchesActionCreators.GetMatches,
    } as any);
    expect(state).toMatchSnapshot();
  });

  it('should perform SetMatches action', () => {
    const state = matchesReducer(initialState, {
      type: MatchesActionCreators.SetMatches,
      payload: [],
    });
    expect(state).toMatchSnapshot();
  });
});
