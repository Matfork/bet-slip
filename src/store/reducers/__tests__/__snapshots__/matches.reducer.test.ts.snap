// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches Reducer should match snapshot 1`] = `
Object {
  "matches": Array [],
}
`;

exports[`Matches Reducer should perform GetMatches action 1`] = `
Object {
  "matches": Array [],
}
`;

exports[`Matches Reducer should perform SetMatches action 1`] = `
Object {
  "matches": Array [],
}
`;
